src/types/related-event.ts

```diff
@@ -0,0 +1,5 @@
+enum RelatedEvent {
+  ENTRIES_CHANGED = 'ENTRIES_CHANGED'
```

Comment:
```suggestion
  RELATED_ENTRIES_CHANGED = 'relatedentrieschanged' 
```
our event convention is lower case